.eslintrc.json

```diff
@@ -31,7 +31,22 @@
         "semi": ["error", "always"],
         "no-console": "error",
         "no-var": "error",
-        "prefer-const": "error"
+        "prefer-const": "error",
```

Comment:
use ' instead of " ?

Reply:
Normally it's already a part of the templates used above